chromakeying.php

```diff
@@ -48,7 +48,7 @@
 				<a class="btn btn--flex" href="javascript:printImage()"><i class="fa fa-print"></i> <span data-l10n="print"></span></a>
 			<?php endif; ?>
 
-			<a class="btn btn--flex" href="./"><i class="fa fa-times"></i> <span data-l10n="close"></span></a>
+			<a class="btn btn--flex" href="javascript:window.history.back()"><i class="fa fa-times"></i> <span data-l10n="close"></span></a>

```

Comment:
this does not work properly here, i need to reload the page to see all photos
(take pic -> open in gallery -> chromakeying -> save -> go back -> pictures missing)

Reply:
must be the url with looks odd

Reply:
once you open a picture in the gallery it adds something like `#&gid=1&pid=6` to the url